auth/role.go

```diff
@@ -23,8 +23,6 @@ var AddonRole = []string{
 	"tagtypes:*",
 	"tasks:get",
 	"tasks.report:*",
-	"volumes:get",
-	"volumes:put",
```

Comment:
Was needed by the "admin" addon which no longer exists.  No reason for addons to work with this API.